test/factories/neighbourhood.rb

```diff
@@ -1,42 +1,86 @@
 # frozen_string_literal: true
 
 FactoryBot.define do
+  factory :neighbourhood_region, class: 'Neighbourhood' do
```

Comment:
Add comment like "Neighbourhoods representing various levels of zoom from most zoomed out to most zoomed in"?